package.json

```diff
@@ -14,7 +14,7 @@
   },
   "devDependencies": {
     "tree-sitter-cli": "^0.16.9",
-    "tree-sitter-javascript": "tree-sitter/tree-sitter-javascript"
+    "tree-sitter-javascript": "github:tree-sitter/tree-sitter-javascript#3d54934"
```

Comment:
Not sure what to do in terms of dependencies and versioning.

Reply:
👍  This makes sense to me.